__tests__/index.test.js

```diff
@@ -87,6 +88,25 @@ describe('GET /api/articles/:article_id', () => {
             expect(msg).toBe('Article not found.')
     });
     })
+    test('GET 200, return the full article including the new comment count feature', () => {
+        return request(app)
+        .get('/api/articles/4')
+        .expect(200)
+        .then(({body}) => {
+            expect(body).toEqual(expect.objectContaining({
+                article_id: 4,
+                title: expect.any(String),
+                topic: expect.any(String),
+                author: expect.any(String),
+                body: expect.any(String),
+                created_at: expect.any(String),
+                votes: expect.any(Number),
+                article_img_url: expect.any(String),
+                comment_count: expect.any(Number),
```

Comment:
Great that you have done a new test but you have made a request to a specific article id so we can and should assert the comment count property is the correct number (not just any number).